mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule PcoApi.Mixfile do
   def project do
     [
       app: :pco_api,
-      version: "0.1.0",
-      elixir: "~> 1.2",
+      version: "0.2.0",
+      elixir: "~> 1.9.2",
```

Comment:
Is there anything that is requiring 1.9?

Reply:
Good call, I got too excited about updating all the things, but I don't think there were any requirements for this.